lib/block_view_validator.go

```diff
@@ -1817,6 +1979,31 @@ func (bav *UtxoView) GetValidatorByPublicKey(validatorPublicKey *PublicKey) (*Va
 	return validatorEntry, nil
 }
 
+func (bav *UtxoView) GetBLSPublicKeyPKIDPairEntryByBLSPublicKey(blsPublicKey *bls.PublicKey) (*BLSPublicKeyPKIDPairEntry, error) {
```

Comment:
It's already contextualized in the function param that it's a lookup by BLS public key. Worth simplifying the function name in that case
```suggestion
func (bav *UtxoView) GetBLSPublicKeyPKIDPairEntry(blsPublicKey *bls.PublicKey) (*BLSPublicKeyPKIDPairEntry, error) {
```